packages/web-components/fast-components/src/__test__/README.md

```diff
@@ -0,0 +1,4 @@
+## Component schema
+
+The `component.schema.json` is used to test against JSON schemas for each component. It should mirror the definition created by [Open UI](https://github.com/WICG/open-ui/blob/master/research/src/schemas/component.schema.json5).
```

Comment:
Could we remove "should". If we are not 100% certain we probably should not document it.  I would suggest "The `component.schema.json` tests against JSON schemas for each component and mirror's the definition created by ..."

Reply:
I used the word "should" because even though this should mirror it, at any point the Open UI definition could change and we could get out of sync.